WireAPI/Sources/WireAPI/Extensions/JSONEncoder+Default.swift

```diff
@@ -24,12 +24,9 @@ extension JSONEncoder {
 
     static var defaultEncoder: JSONEncoder {
         let encoder = JSONEncoder()
+
+        // This is needed so we can deterministically snapshot request bodies.
         encoder.outputFormatting = .sortedKeys
```

Comment:
Can't we put `#if DEBUG` around this line, so that in production we don't waste resources?

Reply:
Good idea, done! bd4ba7fdad6ec9aa2a5708c648af98a3b5cf1cd3